docs/usage/configuration-options.md

```diff
@@ -2302,6 +2302,24 @@ This works because Renovate will add a "renovate/stability-days" pending status
 
 Add to this object if you wish to define rules that apply only to minor updates.
 
+## mode
+
+This configuration option was created primarily for use with Mend's hosted app, but can also be useful for some self-hosted use cases.
+
+It enables a new `silent` mode to allow repos to be scanned for updates _and_ for users to be able to request such updates be opened in PRs _on demand_ through the Mend UI, without needing the Dependency Dashboard issue in the repo.
```

Comment:
I also thought of "on-demand" as an alternative name, but we use the term "silent" in our UI already so sticking to that.